db/models.py

```diff
@@ -1 +1,16 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return f"{self.name}"
```

Comment:
In the future, do not place a single field in an f-string, there is no need to apply string formatting to a CharField. Make this instead:
```suggestion
        return self.name
```